### PR TITLE
revert_changes_from-2.41.0

### PR DESCRIPTION
**Pull request by rasika-chivate**
Revert the changes for PMM: https://jira.percona.com/browse/PMM-8655

### CODE REVIEW

**Review by BupycHuk — APPROVED** [review]

File: docs/get-started/query-analytics.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The dashboard contains three panels:
 - the [Details Panel](#details-panel).
 
 !!! note alert alert-primary ""
-    Query Analytics data retrieval is not instantaneous and can be delayed due to network conditions. In such situations QAN data continues to be gathered and is delivered as soon as the connection is re-established.
+    Query Analytics data retrieval is not instantaneous and can be delayed due to network conditions. In such situations *no data* is reported and a gap appears in the sparkline.
```
Comment (BupycHuk) [inline docs/get-started/query-analytics.md:21]: small amount of data will be gathered(up to 1 hour) and buffered in memory and reported when connection is back.

File: docs/quickstart/index.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+            ```sh
+            apt update
+            apt install -y pmm2-client
+            ```
+
+        3. [Register database node with PMM](../setting-up/client/index.html#register):
+                
+            ```sh
+            pmm-admin config --server-insecure-tls --server-url=https://admin:admin@X.X.X.X:443
+            ```
```
Comment (BupycHuk) [inline docs/quickstart/index.md:86]: these 3 steps are identical for all three installments

File: docs/quickstart/index.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+    1. From the PMM UI, go to **Configuration > PMM Inventory > Add Instance** and choose **PostgreSQL**.
+
+    2. Enter your database credentials on the resulting page.
+
+    3. (Optional) Enter the information on the **Labels** or **Additional Options** section. 
+
+    4. Click **Add Service** at the bottom.
+
+    For detailed information, see [Adding a PostgreSQL database](../setting-up/client/postgresql.md).
```
Comment (BupycHuk) [inline docs/quickstart/index.md:117]: why do we recommend to use UI to add PostgreSQL and MongoDB, but CLI for MySQL

File: docs/release-notes/2.41.0.md [hunk 4]
```diff
@@ -20,29 +20,6 @@ Percona Monitoring and Management (PMM) is an open source database monitoring, m
...
-The offline metric collection introduced in v2.33.0 has now been extended to QAN metrics. Previously, it was limited to Prometheus exporters only.
-
-PMM data is crucial for monitoring and troubleshooting. PMM operates on a client-server architecture, exchanging data between clients (on database servers) and servers (PMM UI interface and data storage). 
-
-Previously, if the PMM client-server connection was interrupted by a network or application issue, the system would stop collecting data, resulting in data collection interruption.
-
-Starting with PMM 2.41.0, if the PMM client and server components encounter connection problems, the QAN data gathered during these periods is stored and transmitted immediately after the connection is restored.
-
-
-### Technical Preview of PMM in HA mode
```
Comment (sleto-it) [inline docs/release-notes/2.41.0.md:34]: Hi.  I do not get why we are removing the section `Technical preview of PMM in HA mode` from the `Release Highlights`.  I thought we wanted to remove the section `Offline metric collection` (related to PMM-8655) and that we wanted to keep the technical preview of the HA mode (as we have documented it in the how-to folder).  Can this be clarified, please.  I think the section `Technical preview of PMM in HA mode` should not be removed.  Thank you

Comment (rasika-chivate) [reply docs/release-notes/2.41.0.md:34]: I have confirmed it with @rnovikovP and he does not want me to include it.

File: docs/quickstart/index.md (hunk 2 above)
Comment (rasika-chivate) [reply docs/quickstart/index.md]: This file has been removed from this branch and will be added in main as its not related to PMM 2.41.0.

File: docs/quickstart/index.md (hunk 3 above)
Comment (rasika-chivate) [reply docs/quickstart/index.md]: Removed this file from this branch as it is not related to this PR.

File: docs/get-started/query-analytics.md (hunk 1 above)
Comment (rasika-chivate) [reply docs/get-started/query-analytics.md:21]: @BupycHuk This has been removed from PMM 2.41.0. Please reaffirm.

Comment (BupycHuk) [reply docs/get-started/query-analytics.md:21]: The message I shared is a current state. we still gather some data and buffer it, so they will survive short network issues. We wanted to extend data buffering for longer period, but had to revert that.

Comment (rasika-chivate) [reply docs/get-started/query-analytics.md:21]: Fixed.

**Pull request merged**